ek9000App/src/devEL3XXX.cpp

```diff
@@ -333,36 +325,27 @@ static long EL36XX_read_record(void* prec) {
 	TerminalDpvt_t* dpvt = static_cast<TerminalDpvt_t*>(pRecord->dpvt);
 
 	/* Check for invalid */
-	if (!dpvt->pterm)
-		return 0;
+	if (!util::DpvtValid(dpvt))
+		return 1;
 
 	/* Lock mutex */
-	int status;
-
-	status = dpvt->pterm->getEK9000IO(MODBUS_READ_INPUT_REGISTERS,
-									  dpvt->pterm->m_inputStart + ((dpvt->channel - 1) * 2), buf, 2);
-	pdo = reinterpret_cast<EL36XXInputPDO_t*>(buf);
-	pRecord->rval = pdo->inp;
-
-	/* Check the overrange and underrange flags */
-	if ((pdo->status & EL36XX_OVERRANGE_MASK) || (pdo->status & EL36XX_UNDERRANGE_MASK)) {
-		recGblSetSevr(pRecord, READ_ALARM, MAJOR_ALARM);
-	}
-
-	/* Set props */
-	pRecord->pact = FALSE;
```

Comment:
Just checking - `pact = FALSE` removal was intentional?

Reply:
My understanding is that `PACT` isn't set to `true` by EPICS implicitly, and is intended to be used with callbacks. Maybe I misunderstood?

Reply:
Yep, that's right

Reply:
The app developer guide seems to confirm this: 
![image](https://github.com/slaclab/epics-ek9000/assets/19717056/ff2c5702-d1f8-415b-91a0-5e9b4ced37a6)

https://epics.anl.gov/base/R3-16/2-docs/AppDevGuide.pdf pg 196-197 has the example synchronous device support module.

I believe the `PACT` here was leftover from when this was using callbacks instead of buffered IO from the device.